demos/multi_material/compositional_buoyancy/compositional_buoyancy.py

```diff
@@ -62,24 +64,24 @@
 # solutions, as in our previous tutorials.
 
 # +
-nx, ny = 40, 40  # Number of cells in x and y directions
-lx, ly = 0.9142, 1  # Domain dimensions in x and y directions
+mesh_elements = (64, 64)  # Number of cells in x and y directions
+domain_dims = (0.9142, 1.0)  # Domain dimensions in x and y directions
 # Rectangle mesh generated via Firedrake
-mesh = RectangleMesh(nx, ny, lx, ly, quadrilateral=True)
-mesh.cartesian = True
-boundary = get_boundary_ids(mesh)
+mesh = RectangleMesh(*mesh_elements, *domain_dims, quadrilateral=True)
+mesh.cartesian = True  # Tag the mesh as Cartesian to inform other G-ADOPT objects
+boundary = get_boundary_ids(mesh)  # Object holding references to mesh boundary IDs
 
-V = VectorFunctionSpace(mesh, "CG", 2)  # Velocity function space (vector)
-W = FunctionSpace(mesh, "CG", 1)  # Pressure function space (scalar)
+V = VectorFunctionSpace(mesh, "Q", 2)  # Velocity function space (vector)
```

Comment:
This convention is different from our previous demos. I'd argue we should be consistent. So perhaps reverting to CG is the least work? The fact that we are using quads is implicit from the mesh right (quadrilateral=True) and I find CG more intuitive.

Reply:
I'm happy with either of these because I don't think we have use DG or DQ in previous tutorials. Perhaps DQ?